File: seed/java-sdk/examples/no-custom-config/src/main/java/com/snippets/Example19.java
package com.snippets;

import com.seed.examples.SeedExamplesClient;
import java.util.Optional;

public class Example19 {
    public static void main(String[] args) {
        SeedExamplesClient client = SeedExamplesClient
            .builder()
            .token("<token>")
            .url("https://api.fern.com")
            .build();

        client.service().refreshToken(Optional.of());
    }
}

File: seed/swift-sdk/client-side-params/with-custom-config/Snippets/Example19.swift
import Foundation
import MyCustomModule

private func main() async throws {
    let client = MyCustomClient(
        baseURL: "https://api.fern.com",
        token: "<token>"
    )

    _ = try await client.service.getconnection(
        connectionId: "connectionId",
        fields: .value("fields")
    )
}

try await main()
